demos/pom.xml

```diff
@@ -163,9 +163,6 @@
         <plugin>
```

Comment:
Where do you see skip set to true? Yes, it should be on by default and problematic modules blacklisted.